doc/whats_new.rst

```diff
@@ -196,6 +196,11 @@ Bug fixes
       :clas:`discriminant_analysis.LinearDiscriminantAnalysis` now returns
       correct results. By `JPFrancoia`_
 
+    - Fixed bug in :class:`linear_model.sgd_fast.SquaredEpsilonInsensitive`
```

Comment:
yes I agree. I would reformulate as fixed in bug in class:`sklearn.linear_model.SGDRegressor` with `loss='squared_epsilon_insensitive'`.